### PR TITLE
Add getRecord to the SDK

### LINKED ISSUE #42: Support record access
The SDK needs to support the ability to access records associated with a folder.

### PR DESCRIPTION
**Pull request by slifty**
This PR adds a `getRecord` function to the sdk's API, allowing SDK users to load records from Permanent.

Resolves #42

### CODE REVIEW

File: src/api/getRecordVo.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  recordId: number,
+  archiveId: number,
+): Promise<RecordVo> => {
+  const queryParams = new URLSearchParams({
+    recordId: `${recordId}`,
+    archiveId: `${archiveId}`,
+  });
+  const response = await makePermanentApiCall(
+    clientConfiguration,
+    `/record/getById?${queryParams.toString()}`,
```
Comment (Fenn-CS) [inline src/api/getRecordVo.ts:31]: The endpoint that supports `version 2` is `/api/record/get` the call structure remains the same.

**Review by Fenn-CS — COMMENTED** [review]
I am not sure if you tested this against a branch locally but please clerify `api/get/getById` does not support version 2 requests yet.

Otherwise this looks great and I would wait for clarification.

File: src/api/getRecordVo.ts (hunk 1 above)
Comment (Fenn-CS) [reply src/api/getRecordVo.ts:31]: Essentially, the endpoint that we have started working on to support `v2` is the one I stated above. 

See : https://github.com/PermanentOrg/back-end/pull/252

Comment (slifty) [reply src/api/getRecordVo.ts:31]: Ahh thank you great catch!  I had the wrong endpoint.

**Review by Fenn-CS — APPROVED** [review]
Great! Thanks @slifty !

File: src/api/getRecordVo.ts (hunk 1 above)
Comment (Fenn-CS) [reply src/api/getRecordVo.ts]: Cool, now looks good and I would mar this as resolved.

**Comment by slifty** [comment]
I added a few little things to this PR because I found some additional fields that were missing in the current File / Record which are needed for the rclone project.  Specifically: `size` and `downloadName`

**Pull request merged**